CARDS-1964: Prems surveys updates

marta- opened the pull request:
* Re-numbered the CPESIC questions
* Changed the titles of OAIP and OED
* Reordered OAIP and OED questions per latest specs
* Added new OAIP question
* Renamed IC (Integrated care) to EDIP (Emergency Department to Inpatient)
* Updated time estimates displayed to the user for filling out their surveys
* Bug fix: Labels of Boolean questions are now displayed in view mode for the CPESIC survey
* Bug cover-up: Removed the `department` column from the clinic dashboard, as `CopyAnswers` fails to copy multivalued answers correctly
* Polishing: Better names for the email notification tasks

marta- committed:
CARDS-1964: Prems surveys updates

marta- committed:
CARDS-1964: Prems surveys updates
CPESIC:
- Fixed variable name in conditionals
- Changed all section labels title case

marta- committed:
CARDS-1964: Prems surveys updates
Bug fix: removed answer options from Boolean questions, as they prevent the labels from being displayed in view mode

marta- committed:
CARDS-1964: Prems surveys updates
Update the id of the answers displayed on the clinic dashboard, to match the new question ids in the surveys

marta- committed:
CARDS-1964: Prems surveys updates
Rename IC to EDIP (this is not the Integrated Care survey, but the survey
for patients who went to Inpatient Care from the Emergency department).

marta- committed:
CARDS-1964: Prems surveys updates
Update the time estimates for filling out the surveys

marta- committed:
CARDS-1964: Prems surveys and clinics updates
Better names for the email notification tasks

sashaandjic commented:
OED and OAIP: goodCPESIC had several issues that I addressed in the spreadsheet. Please make another PR.

marta- commented:
> **OED** and **OAIP**: good
> 
> **CPESIC** had several issues that I addressed in the spreadsheet:
> 
> * fixed different styling in section titles
> * there was a typo in 2 conditionals (they never worked)
> 
> Please make another PR.

Addressed f6e78aa.

sashaandjic commented:
SPESIC form is broken. Some fields in view mode show values instead of labels. Can we redo this PR?

marta- commented:
> CPESIC form is broken. Some fields in view mode show values instead of labels. Can we redo this PR?

 All Boolean questions were imported with AnswerOptions Yes/No, which resulted in unexpected label identification in view mode. The options were removed from the spreadsheet and the survey was re-imported and updated in 8deb869.

marta- commented:
Other changes in the latest commits:
* Bug fix: following the reordering of questions, the wrong answers were referenced to be displayed under "Overall experience" on the clinic dashboard 68c7263.
* The 4th survey was wrongly labeled IC (integrated care) in #1216. It is relabeled as EDIP (Emergency Department to Inpatient) in 13a4f8d.
* Updated the time estimates for filling out the surveys to match the ones proposed in the latest email templates in 465efc3.
* Updated the names of the email notification tasks to match their target clinics that have been changed by #1216 - in 028b3b4.

sashaandjic commented:
Missed this thing in ED - updated spreadsheet by replacing `stay` with `emergency visit` 
![image](https://user-images.githubusercontent.com/23243244/205021201-5f928e77-9036-4f48-b427-52e58e98baf8.png)

**CPES issues:**
- not sure about title "Long Form Inpatient". Since this one is the only Canadian one, not UHN, perhaps "Canadian Inpatient" would be more appropriate
- please note that CAPESIC box is empty even though the form was submitted

![image](https://user-images.githubusercontent.com/23243244/205021613-99836679-8881-4314-973a-ea56d0267918.png)

**UHN Inpatient**
Department and Overall experience is blank
![image](https://user-images.githubusercontent.com/23243244/205022885-d9753e31-f0bf-4be6-b672-9a4d65c95c85.png)

**UHN ED**
blank overall experience
![image](https://user-images.githubusercontent.com/23243244/205023255-3e617f86-a454-4d00-a990-f959c9465085.png)

**UHN ED-IP**
no surveys are created ( I am assuming we are waiting for details on this one)

marta- committed:
CARDS-1964: Prems surveys and clinics updates
- Updated OED intro question text
- Updated the CPES dashboard title
- Bug fix - overall experience is empty on the clinic dashboard: updated CopyAnswers question ids after question reordering
- Bug fix - no surveys created for the ED-IP bundle: updated the questionnaire set node name

marta- commented:
On the clinic dashboard, the "Department" column is expected to be empty (unless the form was created manually) until 1887 is merged. Everything else should have been addressed in 269324f.

sashaandjic commented:
> On the clinic dashboard, the "Department" column is expected to be empty (unless the form was created manually) until 1887 is merged.

@marta- The forms were created manually.  I deleted automatically generated ones and manually created them again in order to test surveys from patient point of view.

marta- committed:
CARDS-1964: Prems surveys and clinics updates
Bug fix -email templates are not included in the distribution: the renamed template folders were not updated in the pom file

marta- committed:
CARDS-1964: Prems surveys and clinics updates
Bug fix - doepartment and overall experience do not show up on the clinic dashboard for CPESIC and OED
Changed the paths of the source questions which were incorrect, as they did not include the section

marta- commented:
> > On the clinic dashboard, the "Department" column is expected to be empty (unless the form was created manually) until 1887 is merged.
> 
> @marta- The forms were created manually. I deleted automatically generated ones and manually created them again in order to test surveys from patient point of view.

Fixed in 6e86bb7.

sashaandjic commented:
![image](https://user-images.githubusercontent.com/23243244/205172856-14c212f7-402a-46ba-a8f9-c8ca14513120.png)

marta- committed:
CARDS-1964: Prems surveys and clinics updates
Temporarily removing the `department` columns in clinic dashboard boxes
since a separate platform bug is causing it to be displayed incorrectly

marta- commented:
@sashaandjic The bug in your screenshot appears to be caused by [CARDS-1970](https://phenotips.atlassian.net/browse/CARDS-1970), which will be fixed separately. Since a clinic dashboard for PREMs is not at all in the requirements and it was just a "nice to have" for our own internal testing, I removed the buggy column in d8e74ce so we can move along with this PR.

sashaandjic reviewed (approved):
tested

During testing I discovered a minor information misplacement on the REVIEW YOUR RESPONSE screen. This will be logged separately.

marta- commented:
> tested
> 
> During testing I discovered a minor information misplacement on the REVIEW YOUR RESPONSE screen. This will be logged separately.

Logged as [CARDS-1971](https://phenotips.atlassian.net/browse/CARDS-1971)
